RoboFile.php

```diff
@@ -577,7 +577,7 @@ public function processChangelog()
             $repo = $release['repo'] ?? 'Codeception';
             $changelog .= sprintf("\n\n### %s %s: %s\n\n", $repo, $release['tag_name'], $release['name']);
 
-            $changelog .= sprintf("Released by [![](%s) %s](%s) on %s",
+            $changelog .= sprintf('Released by [![](%s){:height="16" width="16"} %s](%s) on %s',
```

Comment:
I got the syntax from https://kramdown.gettalong.org/syntax.html#images

> Since additional attributes can be added via span and block IALs, it is possible, for example, to specify image width and height:
> 
> `Here is an inline ![smiley](smiley.png){:height="36px" width="36px"}.`